### PR TITLE
Update Kubernetes to v1.20.11

### PR DESCRIPTION
**Pull request by manuelbuil**
Signed-off-by: Manuel Buil <mbuil@suse.com>
#### Proposed Changes ####

Update Kubernetes

#### Types of Changes ####

Kubernetes

#### Verification ####

`kubectl version -o yaml`

#### Linked Issues ####

* https://github.com/k3s-io/k3s/issues/4005

#### User-Facing Change ####

```release-note
Update Kubernetes to v1.20.11
```

### CODE REVIEW

**Review by briandowns — APPROVED** [review]

**Review by dereknola — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -130,7 +130,7 @@ require (
 	k8s.io/cri-api v1.20.0
 	k8s.io/klog v1.0.0
 	k8s.io/kubectl v1.20.5
```
Comment (brandond) [inline go.mod:132]: The submodules here got incorrectly versioned at some point. Except for the main kubernetes module and klog/yaml/client-go, the ones that are currently at v1.20.5 or v0.20.5 should move to v0.20.11. See the other branches for an example.

**Review by brandond — CHANGES_REQUESTED** [review]
LGTM to push tags. One nit on submodule versions.

File: go.mod (hunk 1 above)
Comment (manuelbuil) [reply go.mod]: I think now they are all on 1.20.11

**Review by brandond — APPROVED** [review]
LGTM!

**Comment by brandond** [comment]
Upstream is building with golang 1.15.15, can you ensure that the golang version gets bumped as well?

**Pull request merged**